src/GafferUIModule/GraphGadgetBinding.cpp

```diff
@@ -217,6 +217,39 @@ void layoutNodes( const GraphLayout &layout, GraphGadget &graph, Gaffer::Set *no
 
 } // namespace
 
+namespace GafferUIModule
+{
+	class ActivePlugsWrapperClassToUseAsFriend
+	{
+	public:
+		static boost::python::tuple activePlugsAndNodesWrapper(
+					const Gaffer::Plug *plug,
+					const Gaffer::Context *context,
+					const IECore::Canceller *canceller
+		)
+		{
+			std::unordered_set<const Gaffer::Plug*> activePlugs;
+			std::unordered_set<const Gaffer::Node*> activeNodes;
+
+			GraphGadget::activePlugsAndNodes( plug, context, canceller, activePlugs, activeNodes );
+
+			boost::python::list activePlugsList;
+			for( const auto &p : activePlugs )
+			{
+				activePlugsList.append( Gaffer::PlugPtr( const_cast<Plug*>( p ) ) );
+			}
+
+			boost::python::list activeNodesList;
+			for( const auto &n : activeNodes )
+			{
+				activeNodesList.append( Gaffer::NodePtr( const_cast<Node*>( n ) ) );
+			}
```

Comment:
Probably need a ScopedGILRelease around this (but not `make_tuple`) for task based computations.

Reply:
Addressed in 5f9175052